Checkpoint2/Program.cs

```diff
@@ -0,0 +1,320 @@
+                        if(movingLeft && board.Grid[columnStart-1,rowStart+1] == "O")
+                        {
+                            //remove enemy that player has jumped
+                            board.RemoveChecker(columnStart-1,rowStart+1);
+                        }
+                        // if player moves right and spot one right and one down is enemy
+                        else if(!movingLeft && board.Grid[columnStart+1,rowStart+1] == "O")
+                        {
+                            //remove enemy that player has jumped
+                            board.RemoveChecker(columnStart+1,rowStart+1);
+                        }
+                        //start grid position is now empty
+                        board.Grid[columnStart,rowStart] = " ";
+                        //To grid positon = selected checkers symbol
+                        board.Grid[columnTo,rowTo] = selectedChecker.Symbol;
+                        //changes selected checkers position to new position
+                        selectedChecker.Position[0] = columnTo;
+                        selectedChecker.Position[1] = rowTo;
+                    }
+                    //if move is not valid let user know
+                    else
+                    {
+                        Console.WriteLine("invalid move");
+                    }
+                }
+                //draws new board before loop repeats or ends
+                board.CreateBoard();
+            }
+            //if all checkers in Checkers list are white then white wins
+            if(board.Checkers.All(x => x.Color == "white"))
+            {
+                Console.WriteLine("White wins!");
+            }
+            //if all checkers in Checkers list are black then black wins
+            else if(board.Checkers.All(x => x.Color == "black"))
+            {
+                Console.WriteLine("Black wins!");
+            }
+        }
+
+        //method checks if user chooses a selectable piece 
+        public bool SelectablePiece(int columnStart, int rowStart)
+        {
+            //piece is outside of bounds of board or selected grid square does not have a piece
+            if ((columnStart > 7 || columnStart < 0 || rowStart > 7 || rowStart < 0) || board.Grid[columnStart,rowStart] == " ")
+                {
+                    //let user know no piece was selected and return false
+                    Console.WriteLine("No piece selected");
+                    return false;
+                }
+            else
+            {
+                return true;
+            }
+        }
+        //if selected piece is white check if move is legal
+        public bool IsLegalWhite(int columnStart, int rowStart,int columnTo, int rowTo)
+        {
+            //checks if piece is moving left
+            bool movingLeft = MovingLeft(columnStart, columnTo);
+            if(//columnTo and rowTo is empty       and piece is not moving to same row and column
```

Comment:
that if condition is a little hard to read. maybe using variable to hold each "thing" you are testing, or having multiple if statements?